dash/testing/browser.py

```diff
@@ -465,6 +448,10 @@ def _get_chrome(self):
                 "safebrowsing.disable_download_protection": True,
             },
         )
+        options.add_argument("--disable-dev-shm-usage")
+        options.add_argument("--no-sandbox")
+        options.add_argument("--disable-gpu")
+        options.add_argument("--remote-debugging-port=9222")
```

Comment:
Why are we adding these options?

Reply:
I can't run chromedriver without those arguments on ubuntu, I thought they would help.

Reply:
OK - seems to still work for me this way on mac, we can keep them.